docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     ports:
       - "8090:8090"
     healthcheck:
-      test: "python3 /sc-machine/scripts/healthcheck.py"
```

Comment:
This will be a breaking change for any docker compose project that uses sc-machine or sc-machine-derived scripts to install python deps. We should make a notice about this change in the CHANGELOG

Reply:
https://github.com/ostis-apps/geometry.ostis/blob/602b4950b9dd6f9b6b370b87454da6b6fd60c4cf/docker-compose.yml#L34
https://github.com/ostis-apps/nika/blob/4aeb36557f286fa281a525dbf960c966e18b2bcc/docker-compose.yml#L40
All our dockerized projects depend on this...